website/docs/cloud-docs/agents/requirements.mdx

```diff
@@ -0,0 +1,42 @@
+---
+page_title: Terraform Cloud Agents - Requirements - Terraform Cloud and Terraform Enterprise
+description: >-
+  Supported operating systems, Terraform versions, hardware requirements, and networking requirements for Terraform Cloud Agents.
+---
+
+# Requirements
```

Comment:
I just realized that we don't really talk about how all agents within a given agent pool _should_ have the same OS and hardware resources available. If they are not homogenous, then run performance could vary wildly. I know you are moving a lot of these docs around but maybe this would be a good time to add a quick mention of this?

Reply:
ahh thank you! I have added a note here to that effect - let me know what you think!

Reply:
Perfect 👌